keras/layers/core.py

```diff
@@ -323,18 +362,20 @@ def set_weights(self, weights):
             weights = weights[nb_param:]
 
     def get_config(self):
-        return {"name": self.__class__.__name__,
-                "layers": [l.get_config() for l in self.layers],
-                "mode": self.mode,
-                "concat_axis": self.concat_axis}
+        config = {"name": self.__class__.__name__,
+                  "layers": [l.get_config() for l in self.layers],
+                  "mode": self.mode,
+                  "concat_axis": self.concat_axis}
+        base_config = super(Merge, self).get_config()
+        return dict(list(base_config.items()) + list(config.items()))
```

Comment:
We might need a more standard way to merge dictionaries (with the latter dict taking precedence). Like a `config` decorator. Or maybe that would introduce too much implicitness.